l5kit/setup.py

```diff
@@ -31,8 +31,9 @@
         "ipywidgets"
     ],
     extras_require={
-        "dev": ["pytest", "mypy", "setuptools", "twine", "wheel", "pytest-cov", "flake8", "black", "isort",
-                "Sphinx", "sphinx-rtd-theme", "recommonmark", "pre-commit"]
+        "dev": ["pytest==5.4.3", "mypy==0.782", "setuptools", "twine", "wheel", "pytest-cov==2.10.0", "flake8==3.8.3",
+                "black==19.10b0", "isort==5.0.4", "Sphinx==3.1.1", "sphinx-rtd-theme==0.5.0", "recommonmark==0.6.0",
```

Comment:
I understand to specify the version of protobuf & black, but why we need to specify the version for `Sphinx`

Reply:
I think in the long run we should specify versions for everything....it avoids breaking changes as the one from isort today